concent_api/core/payments/backends/sci_backend.py

```diff
@@ -58,25 +58,25 @@ def get_list_of_payments(
     return payments_list
 
 
-def validate_that_all_payment_ts_are_younger_than_last_payment_closure_time_if_payment_exists(
+def validate_that_last_closure_time_is_older_than_last_payment(
     requestor_eth_address: str,
     provider_eth_address: str,
-    oldest_payment_ts: Optional[int],
+    youngest_payment_ts: Optional[int],
```

Comment:
How could this be `None`? From my PoV in the current flow it cannot.

Reply:
I think it is `None` during first contact of pair requestor and provider with concent. If there is no previous payment, there is no it's timestamp as well. Do you agree?